package/src/components/Message/MessageSimple/MessageSimple.tsx

```diff
@@ -234,7 +249,26 @@ const MessageSimpleWithContext = <
             {message.pinned ? <MessagePinnedHeader /> : null}
           </View>
 
-          <View style={[styles.contentWrapper, contentWrapper]}>
+          <Swipeable
+            containerStyle={[styles.contentWrapper, contentWrapper]}
+            friction={2}
+            leftThreshold={enableSwipeToReply ? 100 : 0}
```

Comment:
It seems odd that this threshold should be `0` if `enableSwipeToReply` is disabled. Does that prevent swiping from happening ? It sort of worries me to use `Swipeable` at all if we're not really enabling it, perhaps we should consider wrapping the content within it only if we want to have the feature.

Reply:
I thought of it. But I don't think it changes a lot. This check was simply a sanity thing and we can remove it with nothing changing here. If you don't provide `renderLeftActions` there is no way the swipeable would swipe left.

Reply:
It does change the fact that we're not wrapping every message in a component that has some internal state if the feature is not being used. If it is it's needed of course but otherwise not, since `Swipeable` doesn't automatically change to a `View` if it can't reach its threshold. 

Generally, even if the swipe gestures don't work the calculations will be done regardless if the wrapper component is there.